content/en/docs/collector/management.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Management
+weight: 23
+---
+
+Please be sure to review the following documentation as prerequisite:
+
+- [Getting Started][otel-collector-getting-started] to understand how to install
+  the OpenTelemetry collector.
+- [Configuration][otel-collector-configuration] for how to configure the
+  OpenTelemetry collector, setting up telemetry pipelines.
+
+This document describes how you can manage your OpenTelemetry collector
+deployment at scale.
+
```

Comment:
```suggestion
```

Reply:
Hey @mhausenblas. This had been marked as resolved, but the changes were not applied AFAICT. (The sentence was moved to the start of the page.